rop

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+OBSAH_NAME=rop
+OBSAH_DATA=src
+OBSAH_INVENTORY=./inventories/
+export OBSAH_NAME OBSAH_DATA OBSAH_INVENTORY
```

Comment:
We'll need to disable extra vars and add to this list, I opened https://github.com/theforeman/obsah/pull/76 to do so.